### PR TITLE
Try 5.12.2 using alpha4

### PR DESCRIPTION
**Pull request by SrNetoChan**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Trying a different approach for windows follow the latest instructions using conan:
https://github.com/qtwebkit/qtwebkit/wiki/Building-QtWebKit-on-Windows

### CODE REVIEW

**Comment by SrNetoChan** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+# Avoid Xcode
+if [[ $(uname) == Darwin ]]; then
+  PATH=${PREFIX}/bin/xc-avoidance:${PATH}
+fi
```
Comment (gillins) [inline recipe/build.sh:6]: Looks like we should be using `conan` on OSX also: https://github.com/qtwebkit/qtwebkit/wiki/Building-QtWebKit-on-macOS

File: recipe/build.sh [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+if [[ ${HOST} =~ .*linux.* ]]; then
+  # Missing g++ workaround.
+  ln -s ${GXX} g++ || true
+  chmod +x g++
+  export PATH=${PWD}:${PATH}
+fi
+
 qmake
-make -j $CPU_COUNT
```
Comment (gillins) [inline recipe/build.sh:4]: And maybe just calling `cmake` directly on Linux https://github.com/qtwebkit/qtwebkit/wiki/Building-QtWebKit-on-Linux ?

File: recipe/bld.bat [hunk 3]
```diff
@@ -11,6 +11,9 @@ REM      But, use source tarball from Qt Project; projects identical as of 2018-
 
 cd Tools\qt
 
+REM Debug folder containers
+dir
+
```
Comment (gillins) [inline recipe/bld.bat:16]: I think it should be %PYTHON% rather than `py` to conform to conda best practice...

File: recipe/bld.bat [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+cd Tools\qt
+
+REM Debug folder containers
+dir
+
+py %SRC_DIR%\Tools\qt\build-qtwebkit-conan.py ^
+--compiler=msvc ^
+--qt=%PREFIX%\Library\include\qt ^
+--install
```
Comment (gillins) [inline recipe/bld.bat:20]: Seems `build-qtwebkit-conan.py` was added after Alpha4 released (https://github.com/qtwebkit/qtwebkit/commits/qtwebkit-5.212/Tools/qt/build-qtwebkit-conan.py)... Maybe we need to wait for Alpha5...

Comment (SrNetoChan) [reply recipe/bld.bat:20]: Yes, I have found that, the hard way. Either we wait for alpha5, or I go back to the hold recipes. But in those, perl and ruby don't work, not sure why.

**Comment by gillins** [comment]
> Either we wait for alpha5, or I go back to the hold recipes.

Maybe wait? We probably should be waiting for the final release anyway before adding to conda forge although it would be good to know we can build it before it is released...

**Comment by SrNetoChan** [comment]
For testing purposes, we could download master and try creating a recioe that works.

**Comment by gillins** [comment]
Closed by #15

**Pull request closed without merging**